tasks/release.py

```diff
@@ -1328,7 +1329,7 @@ def build_rc(ctx, major_versions="6,7", patch_version=False, k8s_deployments=Fal
     if sys.version_info[0] < 3:
         return Exit(message="Must use Python 3 for this task", code=1)
 
-    gitlab = Gitlab(project_name=GITHUB_REPO_NAME, api_token=get_gitlab_token())
+    repo = get_gitlab_repo()
```

Comment:
```suggestion
    datadog_agent = get_gitlab_repo()
```
GITHUB_REPO_NAME might be unused?